sdk/tables/Azure.Data.Tables/src/TableServiceClient.cs

```diff
@@ -185,7 +185,7 @@ internal TableServiceClient(Uri endpoint, TableSharedKeyPipelinePolicy policy, A
             HttpPipeline pipeline = sasCredential switch
```

Comment:
If we decide to refactor the switch in `TableClient`, it may make sense to apply those changes here as well.